### ISSUE TITLE
can't update to v9.2.0

### ISSUE THREAD

**Issue by ronydebnath** [dimsav/laravel-translatable#523]
Can't upgrade to the version 9.2.0

when i remove `dimsav/laravel-translatable ` from compose.json file and trying to install, it fetches v9.1.0 and due to its dependencies, it shows this error:

```
Problem 1
    - Installation request for dimsav/laravel-translatable ^9.1 -> satisfiable by dimsav/laravel-translatable[v9.1.0].
    - Conclusion: remove laravel/framework v5.7.1
    - Conclusion: don't install laravel/framework v5.7.1
    - dimsav/laravel-translatable dev-master requires illuminate/support 5.6.* -> satisfiable by illuminate/support[v5.6.0, v5.6.1, v5.6.10, v5.6.11, v5.6.12, v5.6.13, v5.6.14, v5.6.15, v5.6.16, v5.6.17, v5.6.19, v5.6.2, v5.6.20, v5.6.21, v5.6.22, v5.6.23, v5.6.24, v5.6.25, v5.6.26, v5.6.27, v5.6.28, v5.6.29, v5.6.3, v5.6.30, v5.6.31, v5.6.32, v5.6.33, v5.6.34, v5.6.35, v5.6.36, v5.6.37, v5.6.38, v5.6.4, v5.6.5, v5.6.6, v5.6.7, v5.6.8, v5.6.9].
    - illuminate/support v5.6.4 conflicts with laravel/framework[v5.7.1].
    - illuminate/support v5.6.5 conflicts with laravel/framework[v5.7.1].
    - illuminate/support v5.6.6 conflicts with laravel/framework[v5.7.1].
    - illuminate/support v5.6.7 conflicts with laravel/framework[v5.7.1].
...
```

if i add `"dimsav/laravel-translatable": "^9.2",` to the composer.json file manually and run `composer update` it shows this error.
```
rony@b08a7bfbec29:/var/www$ composer update
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - The requested package dimsav/laravel-translatable ^9.2 exists as dimsav/laravel-translatable[dev-issue-309, dev-issue-510, dev-master, dev-release-910, v1.0.1, v2.0.0, v3.0.0, v3.0.1, v4.0.0, v4.0.1, v4.1, v4.1.1, v4.2, v4.2.1, v4.3, v4.3.1, v4.3.2, v4.4, v4.5, v5.0, v5.0.1, v5.1, v5.1.1, v5.1.2, v5.2,v5.3, v5.4, v5.5, v5.5.1, v5.6, v5.6.1, v6.0, v6.0.1, v6.1, v7.0, v7.1, v7.2, v7.2.1, v7.3, v8.0, v8.1, v9.0, v9.1.0] but these are rejected by your constraint.
```

**Comment by Gummibeer** [comment]
I got this problem with another package and have no idea why - but take a look in your `~/.composer/config.json`. I never added the package I had problems with there but it was in the config.

**Comment by Gummibeer** [comment]
I've taken a look in your error-message: your composer hasn't the latest versions from packagist. https://packagist.org/packages/dimsav/laravel-translatable

`dev-issue-309, dev-issue-510, dev-master, dev-release-910` - 510 & 910 aren't available and `v9.2.0` is missing in the list.
Try to run `composer clear-cache` (removes all local copies of all ever installed packages in the composer home directory.
otherwise you can run `composer show --all dimsav/laravel-translatable` - it should list all versions available.
If `v9.2.0` isn't in the list I've no more ideas. The bug isn't in this package but composer/packagist and/or your local cache of these. Sometimes packagist/composer need some time to get a new version.

**Comment by ronydebnath** [comment]
Clearing my composer cache files solved my issue. Thanks @Gummibeer
